package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "svd-viewer",
-  "displayName": "SVD Viewer",
+  "name": "peripheral-inspector",
+  "displayName": "Peripheral Inspector",
   "description": "Standalone SVD Viewer extension extracted from cortex-debug",
```

Comment:
Do you want to also update the description to something like: 

Standalone **Peripheral Inspector** extension extracted from cortex-debug

Reply:
Changed